.github/PULL_REQUEST/new_config.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New Config
+about: A new cluster config
+---
+
+Please follow these steps before submitting your PR:
+
+* [ ] If your PR is a work in progress, include [WIP] in its title
+* [ ] Its commits' summaries are reasonably descriptive
+* [ ] Your PR targets the `master` branch
+* [ ] You've included links to relevant issues, if any
+
+* [ ] Add your custom config file to the `conf/` directory
+* [ ] Add your documentation file to the `docs/` directory
+* [ ] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
+* [ ] Add your custom profile to the `README.md` file in the top-level directory
+* [ ] Add your custom profile to the `README.md` file in the top-level directory
```

Comment:
```suggestion

```

Reply:
@drpatelh Sorry I accidently add that commit to the wrong branch, haven't had my coffee yet! See #138 for your comments.